openmdao/drivers/scipy_optimizer.py

```diff
@@ -449,20 +448,38 @@ def run(self):
                               f"than min allowed ({info['min_improve_pct']:.1f}%)."
                         issue_warning(msg, prefix=self.msginfo, category=DerivativesWarning)
 
+        if MPI:
+            comm = MPI.COMM_WORLD
+        else:
+            comm = FakeComm()
         # optimize
         try:
             if opt in _optimizers:
-                result = minimize(self._objfunc, x_init,
-                                  # args=(),
-                                  method=opt,
-                                  jac=jac,
-                                  hess=hess,
-                                  # hessp=None,
-                                  bounds=bounds,
-                                  constraints=constraints,
-                                  tol=self.options['tol'],
-                                  # callback=None,
-                                  options=self.opt_settings)
+                if comm.rank == 0:
+                    result = minimize(self._objfunc, x_init,
```

Comment:
Unless my eyes betray me, the call to minimize is exactly the same in both parts of the if branch.